### ISSUE TITLE
Updates can add duplicate transactions to the database

### ISSUE THREAD

**Issue by jack89roberts** [alan-turing-institute/AIrsenal#355]
E.g. by running this sequence:

 ```shell
> airsenal_update_db --fpl_team_id 123456
```
Then delete rows from transaction table for the most recent gameweek, then

```shell
> airsenal_update_db --fpl_team_id 123456
```

results in two copies of all transactions for ID 123456  in the database (apart from the ones deleted above), i.e. new entries for all transactions are added, not only the ones that were deleted.

**Comment by jack89roberts** [comment]
Currently can't figure out why this isn't the case for a single ID either when running normally, same thing happens without the 456789 team update in the steps above.

**Comment by jack89roberts** [comment]
Should be fixed by #350
